types/html-to-text/html-to-text-tests.ts

```diff
@@ -1,28 +1,24 @@
-import { compile, FormatCallback, htmlToText, HtmlToTextOptions,
-    TagDefinition } from 'html-to-text';
+import { compile, FormatCallback, htmlToText, HtmlToTextOptions, TagDefinition } from 'html-to-text';
```

Comment:
Oh sorry. I run the configured prettier in root folder `npm run prettier -- --write types/html-to-text/**/*.ts` Reverted those changes but I think we should consider formatting it in a separate commit as you suggest.

Reply:
The maintainer is [considering](https://github.com/html-to-text/node-html-to-text/issues/223#issuecomment-862309355) converting to typescript.  It is not in v9 though but it will make this moot if it happens.